.github/patch_release_template.md

```diff
@@ -8,23 +16,25 @@
 * [ ] QA: Update [documentation](https://confluence.owncloud.com/display/OG/Documentation)
 * [ ] QA: Check the translations coming from [Transifex](https://confluence.owncloud.com/display/OG/Online+Updater%2C+Crash+reporter%2C+Transifex#OnlineUpdater,Crashreporter,Transifex-Transifex)
 * [ ] DEV: Check [dependencies](https://confluence.owncloud.com/display/OG/Dependencies) for updates
-* [ ] DEV: Branch off a release branch called e.g. `release-2.x.y`  (or do the release in a `2.x` version branch? DECISION NEEDED)
+* [ ] DEV: Prepare the release in a `X.x` version branch
 * [ ] DEV: bump VERSION.cmake in master to say 2.(x+1).0 unless already done.
 
 ### Build
 
 * [ ] QA: Create [builds](https://confluence.owncloud.com/display/OG/Build+and+Tags#BuildandTags-Sprintbuild) for theme 'ownCloud' and 'testpilotcloud'
 * [ ] QA: Check the new subfolder of [testing download](https://download.owncloud.com/desktop/ownCloud/testing/) if `*tar.xz.asc` files are there. If not follow the [instructions](https://github.com/owncloud/enterprise/wiki/Desktop-Signing-Knowledge)
-* [ ] QA: Add a link to RC1 to https://owncloud.com/beta-testing/#desktop-beta @Marketing
-* [ ] QA: Inform [stakeholders](https://confluence.owncloud.com/display/OG/Marketing+and+Communication#MarketingandCommunication-InformStakeholdersAfterSuccessfulSprintBuild)
+* [ ] Beta/RC Communication (https://confluence.owncloud.com/display/OG/Marketing+and+Communication)
+   * [ ] Website links for beta (needed for the following posts)
+   * [ ] Central post https://central.owncloud.org/tags/c/news/desktop
+   * [ ] All other stakeholders
```

Comment:
specify?

Reply:
The stakeholders are specified in above link https://confluence.owncloud.com/display/OG/Marketing+and+Communication